daal4py/sklearn_patches/__main__.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+#
+# Copyright 2016-2018 Intel Corporation.  All Rights Reserved.
+#
+# The source code contained or described herein and all documents related
+# to the source code ("Material") are owned by Intel Corporation or its
+# suppliers or licensors.  Title to the Material remains with Intel
+# Corporation or its suppliers and licensors.  The Material is protected
+# by worldwide copyright laws and treaty provisions.  No part of the
+# Material may be used, copied, reproduced, modified, published, uploaded,
+# posted, transmitted, distributed, or disclosed in any way without
+# Intel's prior express written permission.
+#
+# No license under any patent, copyright, trade secret or other
+# intellectual property right is granted to or conferred upon you by
+# disclosure or delivery of the Materials, either expressly, by
+# implication, inducement, estoppel or otherwise.  Any license under such
+# intellectual property rights must be express and approved by Intel in
+# writing.
```

Comment:
This license header seems to contradict the ASL license of the daal4py repo.

Reply:
Yes, this should become Apache.